Frontend/voitheia-App/src/app/components/home/home.component.scss

```diff
@@ -21,4 +22,151 @@
     font-size: $xxtra-large;
     font-weight: $bold;
   }
+  
 }
+
+.btn {
+  display: inline-block;
+  font-weight: 400;
+  color: #212529;
+  text-align: center;
+  vertical-align: middle;
+  cursor: pointer;
+  -webkit-user-select: none;
+     -moz-user-select: none;
+      -ms-user-select: none;
+          user-select: none;
+  background-color: transparent;
+  border: 1px solid transparent;
+  padding: 0.375rem 0.75rem;
+  font-size: 1rem;
+  line-height: 1.5;
+  border-radius: 0.25rem;
+  transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
```

Comment:
We should, as mentioned, use the Material Buttons.
All the standard button design should then be done in our material theme so that all the buttons look the same.

Reply:
changed to more generic material buttons. Design yet to be finalized